Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingPass.cpp

```diff
@@ -381,38 +550,6 @@ namespace AZ
                 return;
             }
 
-            RHI::DeviceDispatchRaysItem dispatchRaysItem;
-
-            // calculate thread counts if this is a full screen raytracing pass
-            if (m_passData->m_makeFullscreenPass)
-            {
-                RPI::PassAttachment* outputAttachment = nullptr;
```

Comment:
Why did this move from BuildCommandListInternal to CompileResources?

Reply:
To have it in the same location as the `ComputePass`.
`BuildCommandListInternal()` would be generally possible, but i like it better in the `CompileResources`, and the only reason it was in there is because the RayTracingPass is different from all other passes and used to create the (device) dispatch-item from scratch every frame there.